web_app/src/form.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@font-face {
+	font-family: 'Raleway', sans-serif;
+}
+
+.modal {
+	font-family: 'Raleway';
+	display: none;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0,0,0,0.4);
+}
+.modal-content {
+	display: flex;
+	justify-content: center;
+}
+
+#new-task-form-title {
+	padding-left: 100px;
```

Comment:
Maybe shorthand?? `padding: 10px 100px` here and when `padding-top == padding-bottom && padding-left == padding-right`